catboost/python-package/catboost/core.py

```diff
@@ -946,9 +946,9 @@ def _base_eval_metrics(self, pool, metrics_description, ntree_start, ntree_end,
         metrics_description_list = metrics_description if isinstance(metrics_description, list) else [metrics_description]
         return self._object._base_eval_metrics(pool, metrics_description_list, ntree_start, ntree_end, eval_period, thread_count, result_dir, tmp_dir)
 
-    def _calc_fstr(self, type, pool, thread_count, verbose):
+    def _calc_fstr(self, type, pool, thread_count, verbose, shap_mode="Auto"):
```

Comment:
Let's remove default for `shap_mode` here. It is better to set default value in a single place. I.e. in `get_feature_importance` method.